.github/workflows/push_pr.yaml

```diff
@@ -7,8 +7,8 @@ on:
 
 env:
   # NOTICE that apart from this, the versions in the chart linter matrix needs to be bumped too.
-  LATEST_K8S_VERSION: 'v1.24.4'
-  MINIKUBE_VERSION: 'v1.28.0'
+  LATEST_K8S_VERSION: '1.25.13'
```

Comment:
@xqi-nr please update env variable to latest supported version

Reply:
@juanjjaramillo 1.25.13 is the latest version that does not break the existing tests

Reply:
As discussed offline, please ignore this comment since tests will break